lib/new_relic/agent/instrumentation/logger.rb

```diff
@@ -22,6 +25,6 @@
       chain_instrument NewRelic::Agent::Instrumentation::Logger
     end
 
-    ::NewRelic::Agent.increment_metric("Supportability/Logging/enabled/Ruby/Logger")
+    ::NewRelic::Agent.increment_metric("Supportability/Logging/Ruby/Logger/enabled")
```

Comment:
This now seems like a duplicate of the metric recorded on `lib/new_relic/agent/log_event_aggregator.rb:166`. I like this location, but can't seem to find a path through the `DependencyDetection` class to increment the metric as `disabled` if `depends_on` fails.

Reply:
I like that idea! This metric seems more appropriate in my opinion in this instrumentation class than the aggregator. I was trying to add something like that to the depends_on block with an `unless`, but it felt a bit messy to me: 

```ruby
  unless depends_on { defined?(::Logger) && NewRelic::Agent.config[:'application_logging.enabled'] }
    ::NewRelic::Agent.increment_metric("Supportability/Logging/Ruby/Logger/enabled")
  end
```
What are your thoughts about that approach? Would you prefer the `not_installed` block? I'm open to either.

Reply:
Decided during mobbing to move the #increment_metric call within .depends_on and return false. If this comes up again, we'll consider adding a `not_installed` method to DependencyDetection

Reply:
After testing this code out in practice, we'll never get the disabled metric because if `application_logging.enabled` is false, `instrumentation.logging` is disabled, so the code within `lib/new_relic/agent/instrumentation/logger.rb` will not run. It only runs when the instrumentation is enabled. 

It seems like keeping the metric recording within `NewRelic::Agent::LogEventAggregator` is the safest place after all.

Reply:
Due to the above, I'm going to remove the `::NewRelic::Agent.increment_metric` call within this file.